src/fat.rs

```diff
@@ -519,10 +497,8 @@ impl FatVolume {
                     Cluster::END_OF_FILE => 0xFFFF,
                     _ => new_value.0 as u16,
                 };
-                LittleEndian::write_u16(
-                    &mut blocks[0][this_fat_ent_offset..=this_fat_ent_offset + 1],
-                    entry,
-                );
+                let p_entry: *mut u16 = &mut blocks[0][this_fat_ent_offset] as *mut _ as *mut u16;
+                unsafe { *p_entry = u16::to_le(entry) };
```

Comment:
Couldn't this be unaligned ?

Reply:
this_fat_ent_offset is 2B aligned

Reply:
`Block` is an abstraction over an array of u8s, which has an alignment of 1, no ?

Reply:
reverted